torchvision/models/mobilenet.py

```diff
@@ -50,21 +50,15 @@ def forward(self, x):
 
 
 class MobileNetV2(nn.Module):
-    def __init__(self, num_classes=1000, width_mult=1.0):
+    def __init__(self, inverted_residual_setting, num_classes=1000, width_mult=1.0):
```

Comment:
This is a backwards incompatible change, because users might have been passing `MobileNetV2(100)`, and this will now fail.

But moving it to the end will make it need to have a default value, which implies we will need to add a default value here as well with
```python
if inverted_residual_setting is None:
    inverted_residual_setting = ...
```
I think we should try to keep backwards-compatibility, to avoid breakages on the user land.

Reply:
you are right, i have changed it to be default param